Cargo.toml

```diff
@@ -41,6 +41,9 @@ pbkdf2 = { version = "0.12", features = ["simple"] }
 text-colorizer = "1.0.0"
 log = "0.4"
 fern = "0.6"
+bytes = "1.4.0"
+text-to-png = "0.2.0"
+indexmap = "1.9.3"
```

Comment:
Hi @WarmBeer, I changed all dependencies to use only the minor versions so that you can upgrade patches.

```
bytes = "1.4"
text-to-png = "0.2"
indexmap = "1.9"
```